### PR TITLE
 set up association between different classes

### PR DESCRIPTION
**Pull request by mckent05**
# In this PR, I have done the following:

- Created a `Classroom` class with an instance variable
- Created a relationship between `Classroom` class and `Student` class
- Created a `Book` class with two instance variables, title and author.
- Created a `Rental` class with one instance variable
- Made a relationship between `Person` class and `Rental` class
- Made a relationship between `Book` class and `Rental` class.
- Ensured that I have met all project requirements.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
```
Comment (ElijahTobs) [inline book.rb]: - [ ] To properly implement the `many-to-many` relationship between `Book` and `Rental`, getters and setters for `rentals` should be present

File: person.rb [hunk 2]
```diff
@@ -2,14 +2,15 @@
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rental
```
Comment (ElijahTobs) [inline person.rb]: - [ ] To properly implement the `has-many` relationship between `Person` and `Rental`, getters and setters for `rentals` should be present

File: person.rb [hunk 3]
```diff
@@ -2,14 +2,15 @@
...
   def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @id = Random.rand(1...1000)
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rental = []
   end
 
   def can_use_services?
```
Comment (ElijahTobs) [inline person.rb]: - [ ] A method for adding rentals should be present.

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @mckent05 👋

You've done really well so far! 👍 

## Major Highlights 💯 

README is professional ✔️ 
You've followed the Gitflow rules. ✔️ 
No linter errors ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @mckent05🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**